cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -298,7 +307,8 @@ function getFallback(paymentMethod) {
                     <span class="adyen-checkout__label">Date of birth</span>
                     <input id="dateOfBirthInput" class="adyen-checkout__input" type="date"/>`;
 
-    var fallback = {ach: ach, ratepay: ratepay};
+    var paysafecard = ' ';
+    var fallback = {ach: ach, ratepay: ratepay, paysafecard: paysafecard};
```

Comment:
Regarding paysafecard, if there is no content in the fallback, I would suggest not to include it here.